README.md

```diff
@@ -7,8 +7,48 @@ The AmpliPi design is entirely open source, from the software, to the firmware,
 
 The system is self-hosted on the Raspberry Pi and is privacy-centric. By design, AmpiPi doesn’t listen to you or spy on you — it just plays your audio! The way it should be. An internet connection is only needed for some external streaming sources, such as Pandora. The Python application running on the Pi hosts a mobile-friendly website and REST API used to control the system. The website is built on top of the REST API.
```

Comment:
"privacy-centric", except passwords to streaming services are stored in plaintext on a computer with a widely-known default username and password...

Reply:
I think the meaning of "privacy-centric" here is that unlike other smart home systems that do most of their processing in the cloud (Alexa, etc), the AmpliPi box performs all functionality locally and does not perform any nefarious functions like collecting usage statistics and sending home, etc.